Makefile

```diff
@@ -128,15 +128,12 @@ endif
 
 XGO := $(strip $(if $(XGOOS),GOOS=$(XGOOS)) $(if $(XGOARCH),GOARCH=$(XGOARCH)) $(if $(XHOST_TRIPLE),CC=$(CC_PATH) CXX=$(CXX_PATH)) $(GO))
 
-.DEFAULT_GOAL := all
-.PHONY: all
-all: build test lint
```

Comment:
Wait, which `PHONY` are you worried about? The one on `build` below is necessary, as is the one in the archive Makefile, no?